lxc/storage.go

```diff
@@ -103,12 +108,27 @@ func (c *cmdStorageCreate) command() *cobra.Command {
 }
 
 func (c *cmdStorageCreate) run(cmd *cobra.Command, args []string) error {
+	var stdinData api.StoragePoolPut
```

Comment:
It only looks like .Config field is used from this, but what about Description?

Reply:
It seems like they forgot this case... I'll add it :) thanks

Reply:
fixed